### PR TITLE
Fix the health check header.

### PR DESCRIPTION
**Pull request by xjr1300**
Fix the health check header.

### CODE REVIEW

File: docs/src/backend/15_testing.md [hunk 1]
```diff
@@ -102,10 +102,10 @@ async fn health_check_works() {
         .to_request();
 
     let res = actix_web::test::call_service(&mut app, req).await;
-    
+
     assert!(res.status().is_success());
     assert_eq!(res.status(), StatusCode::OK);
-    let data = res.headers().get("version").and_then(|h| h.to_str().ok());
+    let data = res.headers().get("health-check").and_then(|h| h.to_str().ok());
```
Comment (robertohuertasm) [inline docs/src/backend/15_testing.md:108]: This is correct. As we're using the `version` header in our example. There's an error, though, in [line 42](https://github.com/BcnRust/devbcn-workshop/blob/f768bbf74f693603e166de9f90247b56fbb37fc5/docs/src/backend/15_testing.md#L42).

**Review by robertohuertasm — CHANGES_REQUESTED** [review]
See the comment in the PR review.

File: docs/src/backend/15_testing.md (hunk 1 above)
Comment (kuroyasu67) [reply docs/src/backend/15_testing.md:108]: My apologies.
I realized my mistake when I saw your comment.

**Comment by robertohuertasm** [comment]
@xjr1300 don't close it. If you want to fix the error in line 42 that would be nice. :D

File: docs/src/backend/15_testing.md (hunk 1 above)
Comment (robertohuertasm) [reply docs/src/backend/15_testing.md:108]: Oh, now I see that in ln. 42 the error was indeed intentional. Well closed. Thanks anyway for your time.

**Pull request closed without merging**